### DISCUSSION TITLE
Add a way to get the user's system theme.

### DISCUSSION THREAD

**Discussion by JulienLecoq** [odin-lang/Odin#4245] [Ideas/Requests]
Currently, there is no builtin way (if I'm not mistaken) to get the user's system theme. This is an important feature to have for GUI application to respect the user's preferences.

**Comment by flysand7** [comment]
There are no builtin ways (as in, in the core library) to do any GUI, so I guess there aren't. If you're using a vendor library, that would be a question for those vendor libraries.

**Reply by JulienLecoq** [reply]
Ok 👍🏻
